docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -206,35 +206,53 @@ V&V Checks:
 
 
 .. list-table:: Intrapartum Decision Tree
-  :widths: 3 15 15
+  :widths:  3 5 10 10 15
   :header-rows: 1
 
   * - ID
     - Decision Information 
+    - Data Value 
+    - Source
     - Notes
   * - 0
-    - XX% of simulants have ectopic pregnancies, abortion or miscarriage
-    - These simulants will NOT continue in the model
+    - Incidence of ectopic pregnancies, abortion or miscarriage
+    - get_draws(gbd_round_id=7, location_id=location_id, gbd_id_type='cause_id', gbd_id=[995,374], source='como', measure_id=6, metric_id=3, age_group_id=24, sex_id=2, year_id=2021, decomp_step='iterative')
+    - incidence_c374 + incidence_c995
+    - These simulants will NOT continue in the model. Use the `total population incidence <Total Population Incidence Rate>`_ rate directly from GBD and do not rescale this parameter to susceptible-population incidence rate using condition prevalence.
```

Comment:
Do you know why we need the total population incidence rate here?  I think it probably has to do with [how we'll calculate the probabilities for full term v. partial term](https://vivarium-research.readthedocs.io/en/latest/models/other_models/pregnancy/gbd_2021_closed_cohort/index.html#pregnancy-term-lengths), but I'm not entirely sure.

Reply:
Yeah, I'm not entirely sure either, but the page you linked (specifically [this table's "Notes" column](https://vivarium-research.readthedocs.io/en/latest/models/other_models/pregnancy/gbd_2021_closed_cohort/index.html#id8)) is where I pulled that assumption from.